pages/dkp/kommander/2.2/workspaces/applications/index.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Applications
+title: Applications
+menuWeight: 30
+excerpt: Applications
+---
+This section documents the applications and application types that you can utilize with Kommander.
```

Comment:
```suggestion
This section documents the applications and application types that you can use with Kommander.
```

Reply:
Fixed in both 2.1 and 2.2 (ddc6dea9b5fa3fc04e28ebdc2a608fda6b62bcc8)